src/electionguard/ballot.py

```diff
@@ -573,9 +628,9 @@ def hashed_ballot_nonce(self) -> Optional[ElementModQ]:
         """
 
         if self.nonce is None:
-            log_warning(
-                f"missing nonce for ballot {self.object_id} could not derive from null nonce"
-            )
+            # log_warning(
```

Comment:
Why is this commented out?

Reply:
Because it's generating a warning when the nonce is None, and in fact, it's a completely normal use case for the nonce to be None. That's how every CiphertextAcceptedBallot is.

Reply:
If that's the case, it should be removed not commented. There is no value in commented code.